hack/test-go.sh

```diff
@@ -58,7 +58,7 @@ KUBE_GOVERALLS_BIN=${KUBE_GOVERALLS_BIN:-}
 # Lists of API Versions of each groups that should be tested, groups are
 # separated by comma, lists are separated by semicolon. e.g.,
 # "v1,compute/v1alpha1,experimental/v1alpha2;v1,compute/v2,experimental/v1alpha3"
-KUBE_TEST_API_VERSIONS=${KUBE_TEST_API_VERSIONS:-"v1,extensions/v1beta1,metrics/v1alpha1,federation/v1alpha1;v1,autoscaling/v1,batch/v1,batch/v2alpha1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1"}
+KUBE_TEST_API_VERSIONS=${KUBE_TEST_API_VERSIONS:-"v1,extensions/v1beta1,metrics/v1alpha1,federation/v1alpha1;v1,autoscaling/v1,batch/v1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1;v1,autoscaling/v1,batch/v2alpha1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1"}
```

Comment:
I have a stack of PRs mostly done which fixes the registered package-- one
of the next steps was going to be to replace the testapi package with a
function that lets you set up the real registration stuff however you like.
Unfortunately, it's not going to make it for 1.3. So please do whatever you
have to, but:

1. Unit tests can't get slower than they already are
2. Try not to leave me with something that's really hard to untangle later
:)

On Wed, May 25, 2016 at 1:55 AM, Maciej Szulik <notifications@github.com>
wrote:

> In hack/test-go.sh
> <https://github.com/kubernetes/kubernetes/pull/25737#discussion_r64537418>
> :
>
> > @@ -58,7 +58,7 @@ KUBE_GOVERALLS_BIN=${KUBE_GOVERALLS_BIN:-}
> >  # Lists of API Versions of each groups that should be tested, groups are
> >  # separated by comma, lists are separated by semicolon. e.g.,
> >  # "v1,compute/v1alpha1,experimental/v1alpha2;v1,compute/v2,experimental/v1alpha3"
> > -KUBE_TEST_API_VERSIONS=${KUBE_TEST_API_VERSIONS:-"v1,extensions/v1beta1,metrics/v1alpha1,federation/v1alpha1;v1,autoscaling/v1,batch/v1,batch/v2alpha1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1"}
> > +KUBE_TEST_API_VERSIONS=${KUBE_TEST_API_VERSIONS:-"v1,extensions/v1beta1,metrics/v1alpha1,federation/v1alpha1;v1,autoscaling/v1,batch/v1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1;v1,autoscaling/v1,batch/v2alpha1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1"}
>
> @lavalamp <https://github.com/lavalamp> with the current testapi
> implementation we can't support multiversions in tests 😞 From my
> investigations there are two options: either we add entire group, which as
> you mentioned will significantly increase the tests time (and is only
> temporary solution); or we need to rework testapi which I've started in the
> PR @caesarxuchao <https://github.com/caesarxuchao> linked, but from what
> I've seen it will require quite a lot of changes, since testapi is spread
> all over the code. Personally I'm happy with either approach, although the
> one from this PR is simpler. Let's have this in as is and we can work on
> improving testapi to support multiversions, which will be required at some
> point in time when more groups will support additional versions. Agreed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25737/files/b907135be59b4ac0d16c86d6330e02ae52fd8373#r64537418>
>